### PR TITLE
Further upstream fixes 

### PR DESCRIPTION
**Pull request by mhsaar**

### CODE REVIEW

File: Arrangement_on_surface_2/include/CGAL/Arr_tags.h [hunk 1]
```diff
@@ -68,7 +68,7 @@ struct Get_left_side_category< Traits_, false > {
 
 template < class Traits_ >
 struct Arr_complete_left_side_category {
-public:
+npublic:
```
Comment (sloriot) [inline Arrangement_on_surface_2/include/CGAL/Arr_tags.h]: I'm pretty sure this is not compiling.

File: Arrangement_on_surface_2/include/CGAL/Arr_tags.h [hunk 2]
```diff
@@ -90,7 +90,7 @@ struct Validate_left_side_category< GeometryTraits_2, false > {
   void missing__Left_side_category()
   { 
     T
-      missing__Left_side_category__assuming__Arr_oblivious_side_tag__instead; 
+      missing__Left_side_category__assuming__Arr_oblivious_side_tag__instead __attribute__((unused));
```
Comment (sloriot) [inline Arrangement_on_surface_2/include/CGAL/Arr_tags.h]: I don't think `__attribute__` is defined on MSCV?

File: Nef_2/include/CGAL/Nef_2/PM_const_decorator.h [hunk 3]
```diff
@@ -488,6 +488,10 @@ check_integrity_and_topological_planarity(bool faces) const
   std::size_t e_num = number_of_edges();
   std::size_t c_num = number_of_connected_components() - iso_vert_num;
   std::size_t f_num = number_of_face_cycles() - c_num + 1;
+  /* Suppress unused-variable warnings when assertions are disabled. */
+  (void)v_num;
+  (void)e_num;
+  (void)f_num;
```
Comment (sloriot) [inline Nef_2/include/CGAL/Nef_2/PM_const_decorator.h]: you can use `CGAL_USE` macro instead.

File: Arrangement_on_surface_2/include/CGAL/Arr_tags.h (hunk 2 above)
Comment (lrineau) [inline Arrangement_on_surface_2/include/CGAL/Arr_tags.h]: In `<CGAL/config.h>`, we have a macro `CGAL_UNUSED` that can be used. Its value is `__attribute__ ((__unused__))` for gcc and clang, and empty for other compilers.

File: Arrangement_on_surface_2/include/CGAL/Arr_tags.h (hunk 1 above)
Comment (lrineau) [inline Arrangement_on_surface_2/include/CGAL/Arr_tags.h]: +1 That cannot compile! Please Michael, you have to fix that file.

**Comment by lrineau** [comment]
This PR is waiting for modifications from you. We have entered the code freeze today. If you do not react sufficiently early, this PR will not make it into CGAL-4.10-beta1.

**Comment by lrineau** [comment]
@mhsaar I wrote a message in this discussion four weeks ago, and it remained unanswered.
https://github.com/CGAL/cgal/pull/1797#pullrequestreview-15712239

Can you please acknowledge that you receive those notifications?

**Pull request closed without merging**